File: src/ui/asq-ui/src/app/_models/domain/meeting-summary-dto.ts
import { BaseDomainDto } from './base-domain-dto';
import * as moment from 'moment';

export class MeetingSummaryDto extends BaseDomainDto{
    public title: string = null;
    public description: string = null;
    public meetingStatusId: number = 0;
    public statusDescription: string = null;
    public startDateUtc: moment.Moment = null;
    public estimatedDuration: number = null;
    public timezoneId: number = 0;
    public creationUserUniqueId: string = null;
    public creationUserName: string = null;
    public creationUserSurname: string = null;
    public creationUserUsername: string = null;
    public thumbnailUrl: string = null;
    public creationUserThumbnailUrl: string = null;
    public hasPassed: boolean = null;
    public hasRecordings: boolean = null; 

    constructor(){ super(); }
}

File: src/lib/models/DTO/Domain/MeetingSummaryDto.cs
namespace DTO.Domain
{
    public class MeetingSummaryDto: BaseDomainDto
    {
        public string Title{ get; set; }
        public string Description{ get; set; }
        public short MeetingStatusId{ get; set; }
        public string StatusDescription{ get; set; }
        public System.DateTime StartDateUtc{ get; set; }
        public int EstimatedDuration{ get; set; }
        public int TimezoneId{ get; set; }
        public System.Guid CreationUserUniqueId{ get; set; }
        public string CreationUserName{ get; set; }
        public string CreationUserSurname{ get; set; }
        public string CreationUserUsername{ get; set; }
        public string ThumbnailUrl{ get; set; }
        public string CreationUserThumbnailUrl{ get; set; }
        public bool HasPassed{ get; set; }
        public bool HasRecordings{ get; set; }
    }
}
